### PR TITLE
Fixed failures is few commands.

### PR DESCRIPTION
**Pull request by Pavithra1602**
Fixed failues in --build, --batch and --tmp commands.

Signed-off-by: Pavithra <pavrampu@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by Pavithra1602** [comment]
Logs:

[root@ltc-zz8-lp4 ras]# avocado run sosreport.py
JOB ID     : 83d7876cf1198bc53cf78d3c3b60d65f20d1b51b
JOB LOG    : /home/OpTest/avocado-fvt-wrapper/results/job-2020-10-08T01.08-83d7876/job.log
 (1/6) sosreport.py:Sosreport.test_short:  
PASS (772.22 s)
 (2/6) sosreport.py:Sosreport.test_user:  PASS (195.64 s)
 (3/6) sosreport.py:Sosreport.test_plugins:  PASS (575.76 s)
 (4/6) sosreport.py:Sosreport.test_others:  
PASS (1116.93 s)
 (5/6) sosreport.py:Sosreport.test_archive:  PASS (781.96 s)
 (6/6) sosreport.py:Sosreport.test_PPC:  PASS (102.60 s)
RESULTS    : PASS 6 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/OpTest/avocado-fvt-wrapper/results/job-2020-10-08T01.08-83d7876/results.html
JOB TIME   : 3546.11 s

File: ras/sosreport.py [hunk 1]
```diff
@@ -181,17 +180,15 @@ def test_others(self):
...
         if not os.path.exists(file_name):
             self.is_fail += 1
             self.log.info("--tmp-dir option failed")
 
         dir_name = self.run_cmd_out("sosreport --no-report --batch --build | "
-                                    "grep located | "
-                                    "cut -d':' -f2").strip()
-        sosreport_dir = os.path.join(dir_name, 'sos_reports')
-        if os.listdir(sosreport_dir) != []:
+                                    "grep '/var/tmp/sosreport'").strip()
```
Comment (harish-24) [inline ras/sosreport.py:190]: will it always be `/var/tmp` or it could be `/tmp` sometimes?

Comment (Pavithra1602) [reply ras/sosreport.py:190]: it will be always /var/tmp unless we specify different folder explicitly.

**Comment by Pavithra1602** [comment]
Raised new PR https://github.com/avocado-framework-tests/avocado-misc-tests/pull/1927

**Pull request closed without merging**